python/dgl/graphbolt/graph_storage/csc_sampling_graph.py

```diff
@@ -194,18 +194,26 @@ def in_subgraph(self, nodes: torch.Tensor) -> torch.ScriptObject:
         ), "Nodes cannot have duplicate values."
         return self._c_csc_graph.in_subgraph(nodes)
 
-    def sample_neighbors(self, nodes: torch.Tensor) -> torch.ScriptObject:
+    def sample_neighbors(
+        self, nodes: torch.Tensor, fanout: int
+    ) -> torch.ScriptObject:
         """Sample neighboring edges of the given nodes and return the induced
         subgraph.
 
         Parameters
         ----------
         nodes: torch.Tensor
             IDs of the given seed nodes.
+        fanout: int
+            The number of edges to be sampled for each node. It should be
+            >= 0 or -1. If -1 is given, all neighbors will be selected.
+            Otherwise, it will pick the minimum number of neighbors between
+            the fanout value and the total number of neighbors.
         """
         # Ensure nodes is 1-D tensor.
         assert nodes.dim() == 1, "Nodes should be 1-D tensor."
-        return self._c_csc_graph.sample_neighbors(nodes)
+        assert fanout >= 0 or fanout == -1, "Fanout shoud have value >= 0 or -1"
```

Comment:
just >= -1?

Reply:
shoud -> should

Reply:
To clarify. for the comment it could be `Fanout should have value >= 0 or -1`, for the code, just fanout >= -1.

Reply:
This style can emphasize  -1 is a special value.